tests/inputs/examples.py

```diff
@@ -1,72 +1,89 @@
 }
 
-examples_as_strings = {k: html.unescape(v) for k, v in _example_dict_raw.items() if v}
-examples_as_dicts = {k: json.loads(v) for k, v in examples_as_strings.items() if v}
+ID = "<ID>"
+BOM_PART_RECORD_HISTORY_IDENTITY = "92758"
+
+
+def fill_gaps(dct: Dict[str, Any]):
```

Comment:
This is all ultimately due to some examples being incomplete, which now fail to deserialize following the openapi-common v2 update because they are not marked as nullable in the spec.

This hook fills them in with arbitrary values, such that they can be correctly deserialized and validated

Reply:
The null values have been removed in a subsequent PR - we should re-instate model and API examples in the openapi package and copy those in here

Reply:
There are still some examples that include implicit null values (i.e. yaml fields without values) for non-nullable fields. I've changed the json loads hook to remove these fields from the resulting dict instead of populating them, which means we get back to deserializing to `Unset` and checking that the logic to convert to `None` works as expected.